### ISSUE TITLE
Validation error with crash diagnostic layer

### ISSUE THREAD

**Issue by ziga-lunarg** [LunarG/CrashDiagnosticLayer#165]
When running vkcube with both CDL and VVL enabled this error is reported:

```
Validation Error: [ VUID-VkDebugUtilsMessengerCreateInfoEXT-messageType-parameter ] | MessageID = 0xdf09e3a0
vkCreateDebugUtilsMessengerEXT(): pCreateInfo->messageType has VkDebugUtilsMessageTypeFlagBitsEXT values (VK_DEBUG_UTILS_MESSAGE_TYPE_DEVICE_ADDRESS_BINDING_BIT_EXT) that requires the extensions VK_EXT_device_address_binding_report.
The Vulkan spec states: messageType must be a valid combination of VkDebugUtilsMessageTypeFlagBitsEXT values (https://docs.vulkan.org/spec/latest/chapters/debugging.html#VUID-VkDebugUtilsMessengerCreateInfoEXT-messageType-parameter)
Objects: 1
    [0] VkInstance 0x1e28f8a9a30
```
